ext/ldap/ldap.stub.php

```diff
@@ -4,281 +4,281 @@
 
-/** @param resource $link_identifier */
-function ldap_mod_add($link_identifier, string $dn, array $entry, array $servercontrols = []): bool {}
+/** @param resource $ldap */
+function ldap_mod_add($ldap, string $dn, array $entry, array $controls = []): bool {}
 
 /**
- * @param resource $link_identifier
+ * @param resource $ldap
  * @return resource|false
  */
-function ldap_mod_add_ext($link_identifier, string $dn, array $entry, array $servercontrols = []) {}
+function ldap_mod_add_ext($ldap, string $dn, array $entry, array $controls = []) {}
 
-/** @param resource $link_identifier */
-function ldap_mod_replace($link_identifier, string $dn, array $entry, array $servercontrols = []): bool {}
+/** @param resource $ldap */
+function ldap_mod_replace($ldap, string $dn, array $entry, array $controls = []): bool {}
 
 /**
- * @param resource $link_identifier
+ * @param resource $ldap
  * @alias ldap_mod_replace
  */
-function ldap_modify($link_identifier, string $dn, array $entry, array $servercontrols = []): bool {}
+function ldap_modify($ldap, string $dn, array $entry, array $controls = []): bool {}
 
 /**
- * @param resource $link_identifier
+ * @param resource $ldap
  * @return resource|false
  */
-function ldap_mod_replace_ext($link_identifier, string $dn, array $entry, array $servercontrols = []) {}
+function ldap_mod_replace_ext($ldap, string $dn, array $entry, array $controls = []) {}
 
-/** @param resource $link_identifier */
-function ldap_mod_del($link_identifier, string $dn, array $entry, array $servercontrols = []): bool {}
+/** @param resource $ldap */
+function ldap_mod_del($ldap, string $dn, array $entry, array $controls = []): bool {}
 
 /**
- * @param resource $link_identifier
+ * @param resource $ldap
  * @return resource|false
  */
-function ldap_mod_del_ext($link_identifier, string $dn, array $entry, array $servercontrols = []) {}
+function ldap_mod_del_ext($ldap, string $dn, array $entry, array $controls = []) {}
 
-/** @param resource $link */
-function ldap_errno($link): int {}
+/** @param resource $ldap */
+function ldap_errno($ldap): int {}
 
-/** @param resource $link */
-function ldap_error($link): string {}
+/** @param resource $ldap */
+function ldap_error($ldap): string {}
 
 function ldap_err2str(int $errno): string {}
 
-/** @param resource $link_identifier */
-function ldap_compare($link_identifier, string $dn, string $attribute, string $value, array $servercontrols = []): bool|int {}
+/** @param resource $ldap */
+function ldap_compare($ldap, string $dn, string $attribute, string $value, array $controls = []): bool|int {}
 
 
 #ifdef LDAP_CONTROL_PAGEDRESULTS
 /**
- * @param resource $link
+ * @param resource $ldap
  * @deprecated since 7.4
  */
-function ldap_control_paged_result($link, int $pagesize, bool $iscritical = false, string $cookie = ""): bool {}
+function ldap_control_paged_result($ldap, int $pagesize, bool $iscritical = false, string $cookie = ""): bool {}
```

Comment:
```suggestion
function ldap_control_paged_result($ldap, int $page_size, bool $critical = false, string $cookie = ""): bool {}
```
Could just `$size` work? Nevertheless, using the `page_` prefix is perfectly fine for me.
`$is_critical` would be ok for me as well, but I think there's an agreement that we don't use the `is_` prefix

Reply:
Actually this function is deprecated since PHP-7.4. Can it be removed in PHP-8?

Reply:
Yes, I think so.

Reply:
Yes, it should be fine to drop. It looks like another deprecated function (ldap_sort) did get dropped already. I'd suggest landing that separately from this PR (just directly commit it with a note in UPGRADING).

Reply:
Ok, I will do that.

Reply:
As we should land this today to ensure the removal goes into PHP 8.0, I've put up #6235. Does that look reasonable?